pkgs/tools/typesetting/lowdown/default.nix

```diff
@@ -17,6 +17,15 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ which ]
     ++ lib.optionals stdenv.isDarwin [ fixDarwinDylibNames ];
 
+  patches = [
+    # for darwin
+    # https://github.com/kristapsdz/lowdown/pull/88
+    (fetchpatch {
+      url = "https://github.com/kristapsdz/lowdown/commit/f41ffde6e86e7adaed6bdf5ecf0e0e2a11eb9bd9.diff";
```

Comment:
That patch should be conditional to darwin, as it effectively breaks compilation with clang on non darwin, since it assumes clang implies cctools ld.

Apart from that it doesn't even seem to work on darwin?